source/routing/index.md

```diff
@@ -30,24 +30,3 @@ or more of the following:
 2. Update a controller so that it represents a particular model.
 3. Change the template on screen, or place a new template into an
    existing outlet.
-
-###Logging Route Changes
```

Comment:
This is the first page that people learn about routes, and I think it's purpose is to introduce the subject at a high level and get them ready to dive into all of the details in the following pages. When you're learning about a new subject, debugging is only useful to know about after you've learned about the major important concepts, not before. Terrible analogy: It's like teaching somebody how to escape from a crashing airplane before they've learned how to fly it. 

In the Templates section, one of the last guides is Development Helpers, which is essentially the debugging tools for templates. Similarly, we could add a section called Debugging Routes or something to the end of the Routing section that included http://guides.emberjs.com/v2.0.0/configuring-ember/debugging/#toc_log-router-transitions and info from Ember Inspector... But currently those logging config options are all grouped together, as is Ember Inspector, and if we want to break them out into their relevant sections I suggest we open another issue for doing that so we can discuss it for all of the Guides, instead of just for Routing.